locator/types.hh

```diff
@@ -0,0 +1,32 @@
+/*
+ *
+ * Modified by ScyllaDB
+ * Copyright (C) 2022-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: (AGPL-3.0-or-later and Apache-2.0)
+ */
+
+#pragma once
+
+#include <seastar/core/sstring.hh>
+
+#include "inet_address_vectors.hh"
```

Comment:
This one looks unrelated

Reply:
hmm, I think I just copied it from token_metadata.hh, but including gms/inet_address.hh should be sufficient

Reply:
it was needed for inet_address_vector_replica_set in topology.hh, but it should indeed be included in topology.hh, not in types.hh